tests/sentry/search/events/builder/test_metrics.py

```diff
@@ -2857,80 +2863,87 @@ def _test_user_misery(
         threshold = 300
         field = f"user_misery({threshold})"
         query_s = "transaction.duration:>=10"
-        spec = OnDemandMetricSpec(field=field, query=query_s, spec_type=MetricSpecType.SIMPLE_QUERY)
+        # You can only query this function if you have the feature flag for the org
+        with Feature(SPEC_VERSION_TWO_FLAG):
```

Comment:
We're right shifting this block.
Reviewing this block with `Hide whitespace` is easier:
<img width="204" alt="image" src="https://github.com/getsentry/sentry/assets/44410/fdc888d1-cbe8-4fe0-8f7d-aa670165709d">